moped-editor/src/components/GridTable/GridTable.js

```diff
@@ -60,15 +60,42 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
+/**
+ * Note: also used in NavigationSearchInput
```

Comment:
because im using this function in NavigationSearchInput, I had to pull it out of the GridTable component so I could export it.

Reply:
This is rad! 👍